components/scripts/lib/cli-parsers/common.m4

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env bash
 
-# ARG_OPTIONAL_BOOLEAN([enable-gradle-enterprise],[e],[])
+# ARG_OPTIONAL_SINGLE([gradle-enterprise-server],[s],[],[])
+# ARG_OPTIONAL_SINGLE([git-repo],[r],[])
 # ARG_OPTIONAL_SINGLE([git-branch],[b],[])
 # ARG_OPTIONAL_SINGLE([git-commit-id],[c],[])
 # ARG_OPTIONAL_SINGLE([git-options],[o],[])
-# ARG_OPTIONAL_SINGLE([gradle-enterprise-server],[s],[],[])
-# ARG_OPTIONAL_SINGLE([git-repo],[r],[])
 # ARG_OPTIONAL_SINGLE([project-dir],[p],[],[])
+# ARG_OPTIONAL_SINGLE([args],[a],[])
+# ARG_OPTIONAL_BOOLEAN([enable-gradle-enterprise],[e],[])
 # ARG_OPTIONAL_BOOLEAN([interactive],[i],[],[off])
 # ARG_OPTIONAL_BOOLEAN([debug],[],[],[off])
-# ARG_OPTIONAL_SINGLE([args],[a],[])
 # ARGBASH_GO()
 # needed because of Argbash --> m4_ignore([
 ### START OF CODE GENERATED BY Argbash v2.10.0 one line above ###
```

Comment:
Could `ARG_VERSION` be moved to this script instead of being defined in each sub-script?

Reply:
I have a vague recollection of trying to put that here and it not working, but that was quite a long time ago. I think it's worth trying it again.

Reply:
I just tried this, and unfortunately, moving `ARG_VERSION` to the common template breaks the version CLI argument. I also tried moving `ARG_HELP` up, but that broke the help CLI argument.  So unfortunately, these need to stay in the individual experiment CLI parser templates.

Reply:
Got it. Thanks for trying.